### PR TITLE
Add Steps to lint in Readme

### PR DESCRIPTION
**Pull request by Ayush0730**
## What is the change?
Added steps to lint in readme as well as in Contributing.md

## Related issue?
close: #613 

## How was it tested?
Remove this line and tell us how the code was tested.

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [] Did you lint your code before making the Pull Request?
- [] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video:
Add a screenshot or demo video if appropriate.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,6 +114,22 @@ http://localhost:3000/
...
+```
+npm run lint
+```
+- If some files have changed by above command, you have to repeat Step 4
+- Note : If you get error "eslint is not recognised", then you have to install eslint by using following command
+```
+npm install -g eslint 
+```
+<br/>
+
```
Comment (Pranav016) [inline README.md:132]: I think this `Note` should come before or after this line `Now you are all set to use this project.` , I feel it shouldn't be before the EJS template line.
image

**Review by Pranav016 — CHANGES_REQUESTED** [review]
One change otherwise the content of the PR looks good 👍

File: README.md (hunk 1 above)
Comment (Ayush0730) [reply README.md:132]: Okay

**Comment by Pranav016** [comment]
/gssoc/

**Comment by Pranav016** [comment]
/level1/

**Pull request merged**